R/a11y_panel.R

```diff
@@ -0,0 +1,250 @@
+      "Chart tooltips are not compativle with screen reader use.",
+      "Some decorative images are not labelled appropriately as yet.",
+      "Some links are not appropriately labelled."
+    )) {
+  # Validate inputs
+  date_tested <- validate_date(date_tested)
+  date_prepared <- validate_date(date_prepared)
+  date_reviewed <- validate_date(date_reviewed)
+  date_template_reviewed <- validate_date(date_template_reviewed)
+  validate_dashboard_url(dashboard_url)
+  if (!is_valid_repo_url(repo_url)) {
+    stop(repo_url, " is not a valid repository url")
+  }
+  shiny::tags$div(
+    "Accessibility",
+    shiny::tags$div(
+      shiny::tags$h1(paste0("Accessibility statement for ", dashboard_title)),
+      shiny::tags$p(
+        "This accessibility statement applies to the",
+        dashboard_url,
+        "website. This website is run by the ",
+        external_link(
+          href = "https://www.gov.uk/government/organisations/department-for-education",
+          "Department for Education (DfE)"
+        ),
+        ".",
+        "This statement does not cover any other services run by the Department for Education ",
+        "(DfE) or GOV.UK."
+      ),
+      shiny::tags$h2("How you should be able to use this website"),
+      shiny::tags$p(
+        "We want as many people as possible to be able to use this website. You should be able to:"
+      ),
+      shiny::tags$div(tags$ul(
+        shiny::tags$li(
+          "change colours, contrast levels and fonts using browser or device settings"
+        ),
+        shiny::tags$li(
+          "zoom in up to 400% without the text spilling off the screen"
+        ),
+        shiny::tags$li(
+          "navigate most of the website using a keyboard or speech recognition software"
+        ),
+        shiny::tags$li(
+          "listen to most of the website using a screen reader
+                    (including the most recent versions of JAWS, NVDA and VoiceOver)"
+        )
+      )),
+      shiny::tags$p("We've also made the website text as simple as possible to understand."),
+      shiny::tags$p(
+        external_link(href = "https://mcmw.abilitynet.org.uk/", "AbilityNet"),
+        " has advice on making your device easier to use if you have a disability."
+      ),
+      shiny::tags$h2("How accessible this website is"),
+      if (all(is.na(non_accessible_components))) {
+        shiny::tags$p("This website is fully compliant with accessibility standards.")
+      } else {
+        shiny::tagList(
+          shiny::tags$p("We know some parts of this website are not fully accessible:"),
+          shiny::tags$div(tags$ul(
+            tagList(lapply(non_accessible_components, shiny::tags$li))
+          ))
+        )
+      },
+      shiny::tags$h2("Feedback and contact information"),
+      shiny::tags$p(
+        "If you need information on this website in a different format please see the parent",
```

Comment:
This might be a proper faff but is there any way to name the parent publication specifically here? It seems like extra work for the user to have to go looking

Reply:
Done

Reply:
Not sure this is fully appearing / pulling through? Think you might need to wrap both parts in a single div as only the second half is showing?

![image](https://github.com/user-attachments/assets/33c40c4c-37db-4cb2-9418-548529036911)

Reply:
Ah yeah, I've added a tagList around the two paragraphs now so they both pull through.